spec.html

```diff
@@ -32195,21 +32352,21 @@ <h1>
           <dt>description</dt>
           <dd>
             It converts _t_ from UTC to local time.
-            The local political rules for standard time and daylight saving time in effect at _t_ should be used to determine the result in the way specified in this section.
+            The local political rules for standard time and daylight saving time in effect at _t_ should be (and, for time zone aware implementations, must be) used to determine the result in the way specified in this section.
```

Comment:
This looks like it's a requirement on certain implementations, but that's kind of confusing - this is more properly part of the 
_definition_ of "time zone aware", right? (Ditto all other uses of similar phrases elsewhere.)

Reply:
👍  I removed the new text in this line here, and the same line in the `UTC` abstract operation.  It's now covered in the clarified & expanded definition of "time zone aware".